apps/style/netsim/style.scss

```diff
@@ -510,78 +564,73 @@ svg {
         margin: 0 1em 10px 1em;
       }
     }
+  }
 
-    .netsim_my_device_tab {
-      .netsim_encoding_selector {
-        div.hiddenControl {
-          display: none;
-        }
+  .netsim-encoding-selector {
+    div.hidden-control {
+      display: none;
+    }
 
-        input[type="checkbox"] {
-          vertical-align: baseline;
-          margin: 0;
-        }
+    input[type="checkbox"] {
+      vertical-align: baseline;
+      margin: 0;
+    }
 
-        label {
-          display: inline;
-          margin-left: 0.25em;
-        }
-      }
+    label {
+      display: inline;
+      margin-left: 0.25em;
     }
+  }
 
-    .netsim_router_tab {
-      .netsim_router_log {
-        max-height: 200px;
-        overflow-y: scroll;
+  .netsim-router-log {
+    max-height: 200px;
+    overflow-y: scroll;
 
-        table {
-          th, td {
-            font-size: 8pt;
-          }
-        }
+    table {
+      th, td {
+        font-size: 8pt;
       }
     }
+  }
 
-    .netsim_dns_tab {
-      .dns_mode_control {
-        input[type="radio"] {
-          vertical-align: middle;
-          margin: 0;
-        }
+  .dns-mode-control {
+    input[type="radio"] {
+      vertical-align: middle;
+      margin: 0;
+    }
 
-        label {
-          display: inline;
-          margin-left: 1em;
-        }
-      }
+    label {
+      display: inline;
+      margin-left: 1em;
+    }
+  }
 
-      .netsim_dns_table {
-        tr.localNode {
-          font-weight: bold;
-          background-color: lightcyan;
-        }
+  .netsim-dns-table {
+    tr.local-node {
+      font-weight: bold;
+      background-color: $own_node_background_color;
+    }
 
-        tr.dnsNode {
-          color: white;
-          background-color: darkorange;
-        }
-      }
+    tr.dns-node {
+      color: $white;
+      background-color: $code_org_web_orange;
+    }
+  }
 
-      .dns_notes {
-        div {
-          textarea {
-            resize: none;
-            width: 100%;
-            height: 100px;
-          }
-          padding-right: 14px;
-        }
+  .dns-notes {
+    div {
+      textarea {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Selector should have depth of applicability no greater than 2, but was 4